utils.py

```diff
@@ -0,0 +1,114 @@
+from files import tasks
+from decorators import view_decorator, create_decorator, timer_decorator
+
+def gen_id(tasks):
+    if not len(tasks):
+        return 1
+    return max(tasks.keys()) + 1
+
+def save_task(tasks):
+    with open("tasks.txt", "w") as file:
+        file.write(tasks_to_strings(tasks))
+
+def tasks_to_strings(tasks):
+    result = ""
+    for task_id, task_values in tasks.items():
+        result += f"{task_id}: {task_values['title']} | {task_values['desc']} | {task_values['priority']} | {task_values['status']}\n"
+    return result
+
+def priority_def():
+    try:
+           while True:
+            priority = int(input("Выберите приоритет задачи (1 - low, 2 - middle, 3 - high): "))
+            if priority == 1:
+                return "low"
+            elif priority == 2:
+                return "middle"
+            elif priority == 3:
+                return "high"
+            else:
+              print("Пожалуйста, введите цифру от 1 до 3.")
+    except ValueError:
+         print("Пожалуйста, введите допустимое число.")
+         save_task(priority)
+
+def status_def():
+     while True:
+        try:
+            status = int(input("Выберите статус важности задачи (1 - low, 2 - middle, 3 - high): "))
+            if status == 1:
+                return "low"
+            elif status == 2:
+                return "middle"
+            elif status == 3:
+                return "high"
+            else:
+                print("Пожалуйста, введите цифру от 1 до 3.")
+        except ValueError:
+            print("Пожалуйста, введите допустимое число.")
+   
+def create_task(task_id):
+            title = input("Введи название: ")
+            desc = input("Введи описание: ")
+            priority = priority_def()
+            status = status_def()
+            save_task(tasks)
+            return {task_id: {"title": title, "desc": desc, "priority": priority, "status": status}}
+
+@view_decorator
+@timer_decorator
+def view_task():
+     with open("tasks.txt", "r") as file:
+          content = file.read()
+          print(content)
+
+@timer_decorator
+def delete_task():
+     view_task()
+     try:
```

Comment:
I think this is the third time I've put all the logic in the `try` block.
Searching for a certain value in a certain collection and handling an error if the desired value is not present seems to me to be a dedicated function.